test/e2e-tests.sh

```diff
@@ -36,23 +42,33 @@ function knative_setup() {
   # Install the latest Knative/eventing in the current cluster
   echo ">> Starting Knative Eventing"
   echo "Installing Knative Eventing"
-  ko apply -f config/ || return 1
+  ko apply -f ${EVENTING_CONFIG} || return 1
   wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
 
   echo "Installing In-Memory ClusterChannelProvisioner"
-  ko apply -f config/provisioners/in-memory-channel/in-memory-channel.yaml || return 1
+  ko apply -f ${IN_MEMORY_CHANNEL_CONFIG} || return 1
   wait_until_pods_running knative-eventing || fail_test "Failed to install the In-Memory ClusterChannelProvisioner"
+
+  echo "Installing GCPPubSub ClusterChannelProvisioner"
+  kubectl -n knative-eventing create secret generic gcppubsub-channel-key --from-file=key.json=${GOOGLE_APPLICATION_CREDENTIALS}
+  E2E_CLUSTER_PROJECT="$(gcloud config get-value project)"
```

Comment:
It would be nice to make this widely available in a future enhancement PR, as this is used in the helper script as well:

https://github.com/knative/test-infra/blob/ad06951c5e236054781eeaacffed25d36abb2cbe/scripts/e2e-tests.sh#L187

Reply:
Create a [PR](https://github.com/knative/test-infra/pull/692) to add a global flag, and added a `TODO` here.

Reply:
Thanks for that PR!